docs/pages/intro/what-is-compositor.md

```diff
@@ -1,8 +1,13 @@
 # What is Live Compositor?
 
-LiveCompositor is a media server for real-time, low latency, programmable video and audio mixing.
+Live Compositor is an engine for applying effects to videos and for combining multiple videos together.
+On a very basic level, it is an application which exposes an HTTP API.
+The API allows you to specify where to get input videos, and how to modify and compose them together.
+The resulting outputs can then be written to a file or streamed to a separate service.
+
+## The TypeScript SDK
+
+The TypeScript SDK is the recommended way to start using the compositor right now.
+It is a library which allows controlling how the compositor manipulates videos in a way which is simpler to wrap your head around than raw HTTP requests.
```

Comment:
```suggestion
It is a library that allows you to control how the compositor manipulates videos with React components.
This approach should be very intuitive for anyone with a web development background and simpler to wrap your head around than raw HTTP requests.
```

Reply:
Also please avoid adding line breaks at the end of the sentence in markdown. Either write everything in one line or add line breaks based on line length. This approach is causing the markdown structure to look very different than what will be rendered on the website.